docs/advanced.md

```diff
@@ -44,6 +45,48 @@ which can be used to show custom page numbers or add other elements to the UI.
 - `step`: (number) the current visible step in an Appear or Step component
 - Each slide includes a `meta` object with a `notes` field when the Notes component is used within a slide
 
+#### Example
+
+The example below will display the current slide out of the total amount of slides.
+
+```js
+import React, { useEffect, useRef, useState } from 'react'
+import { css } from '@emotion/core'
+
+function AtTheBottomCenter ({ children }) {
```

Comment:
I think this whole example would be simpler to read without the additional window logic that could be handled with CSS.

Reply:
Ah, yes I do agree it would be simpler if I removed that bit of logic. I should be able to make the changes at some point this weekend. 😄